### PR TITLE
Added pagnation to user playlists

### PR DESCRIPTION
**Pull request by Peterkmoss**
Added some simple pagnation to at least scroll through your playlist. I don't know if this is the best way to implement it but it seems to work fine from my own testing. Please excuse if my Rust is "rusty", it's my first Rust project and I don't know if there is need for memory optimization for this change to not leak memory.

### CODE REVIEW

File: src/handlers/track_table.rs [hunk 1]
```diff
@@ -105,7 +105,19 @@ pub fn handler(key: Key, app: &mut App) {
...
                 Some(context) => match context {
-                    TrackTableContext::MyPlaylists => {}
+                    TrackTableContext::MyPlaylists => {
+                        if let (Some(playlists), Some(selected_playlist_index)) =
+                            (&app.playlists, &app.selected_playlist_index)
+                        {
+                            if let Some(selected_playlist) =
+                                playlists.items.get(selected_playlist_index.to_owned())
+                            {
+                                app.playlist_offset += 20;
```
Comment (Rigellute) [inline src/handlers/track_table.rs]: Instead of hardcoding `20`, please use `self.large_search_limit`

Comment (Peterkmoss) [reply src/handlers/track_table.rs]: I see, of course - I have fixed it to use the `large_search_limit` from app.

**Comment by Peterkmoss** [comment]
I spotted a problem with the navigation when paging so I added a check to see if the new table should be added to the navigation stack.

**Comment by Rigellute** [comment]
Thanks for again for the contribution

**Comment by Rigellute** [comment]
@all-contributors please add @Peterkmoss for code

**Pull request merged**